.travis.yml

```diff
@@ -58,6 +58,19 @@ jobs:
         - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --config ./audit-ci.json; fi
         - npm run lint
         - sudo env "PATH=$PATH" npm test
+    - stage: 'linting and formatting checks'
```

Comment:
Can you please add "firmware" to the name of this stage? Just to distinguish it from the linting that we do for the server code.

Reply:
I added this in my latest commit